mysql-test/suite/innodb/r/stat_tables.result

```diff
@@ -100,4 +100,19 @@ SET GLOBAL innodb_stats_persistent=@save_persistent;
 DROP TABLE t1;
 CREATE TABLE t1 (c1 INT) ENGINE=InnoDB STATS_PERSISTENT 1;
 DROP TABLE t1;
+#
+# MDEV-32667 dict_stats_save_index_stat() reads uninitialized
+#              index->stats_error_printed
+#
+CREATE TABLE t (a INT) ENGINE=InnoDB STATS_PERSISTENT=1 STATS_AUTO_RECALC=1;
+BEGIN;
+SELECT COUNT(*)>=0 FROM mysql.innodb_index_stats LOCK IN SHARE MODE;
+COUNT(*)>=0
+1
+INSERT INTO t VALUES(0),(0);
+SELECT sleep(5);
+sleep(5)
+0
+COMMIT;
+DROP TABLE t;
```

Comment:
I don’t think we want to add this test case. Any `sleep` should ideally be forbidden in the regression test suite. It makes sense to add this test in #3616, but here I think we had better remove the test case to reduce unnecessary delay on our CI systems.